README.adoc

```diff
@@ -14,20 +14,15 @@ This repository uses https://github.com/golang/go/wiki/Modules[Go modules]. You
 
 == Setting Up and Connecting Host and Member Clusters
```

Comment:
From my point of view, this whole section is outdated, and we can drop it. If there is someone who needs to configure host & member operator, the (s)he should use https://github.com/codeready-toolchain/toolchain-e2e/blob/master/dev_install.adoc 
WDYT @xcoulon ?

Reply:
A reminder :-)  You can remove the whole ` == Setting Up and Connecting Host and Member Clusters` part. Either now or in the next PR - as you wish :wink:

Reply:
And replace it with a note probably then?